girder/utility/server.py

```diff
@@ -18,16 +18,34 @@
 ###############################################################################
 
 import cherrypy
+import mako
 import mimetypes
 import os
 import six
 
 import girder.events
 from girder import constants, logprint
+from girder.api.describe import API_VERSION
 from girder.utility import plugin_utilities, model_importer
 from girder.utility import config
 from . import webroot
 
+# TODO make a pretty error page someday.
+_errorTemplate = """
+<h1>${status}</h1>
+<p>${message}</p>
+<p><i>Girder server version ${version}</i></p>
```

Comment:
Yeah, it's already exposed through the web API and the front page of the web GUI.